README.md

```diff
@@ -388,7 +392,20 @@ endpoints:
       - "[STATUS] == 200"
 ```
 
+Here's an example with the client using a proxy:
+
+```yaml
+endpoints:
+  - name: website
+    url: "https://twin.sh/health"
+    client:
+      proxy-url: http://proxy.example.com:8080
+    conditions:
+      - "[STATUS] == 200"
+```
+
```

Comment:
I'd rather this be added in the FAQ section as opposed to here.

Also, just out of curiosity, is authentication supported? Not asking to add support for it, I'm just curious as to whether specifying credentials in the URL would work.

Reply:
@TwiN Done. See [`877c09a` (#668)](https://github.com/TwiN/gatus/pull/668/commits/877c09aed8a37fee27048d49153cf65dfb543e98)

Reply:
@TwiN Any feedbacks?